apps/mark/frontend/src/components/contest_screen_layout.tsx

```diff
@@ -128,5 +23,4 @@ export const ChoicesGrid = styled.div.attrs({
   display: grid;
   grid-auto-rows: minmax(auto, 1fr);
   grid-gap: 0.5rem;
-  margin-bottom: 1rem;
```

Comment:
No longer needed - was used as a spacer to account for the scroll container shadow - handled within `WithScrollButtons` now